src/Microsoft.AspNetCore.Mvc.Razor/DependencyInjection/MvcRazorMvcCoreBuilderExtensions.cs

```diff
@@ -26,10 +29,19 @@ public static IMvcCoreBuilder AddRazorViewEngine(this IMvcCoreBuilder builder)
             }
 
             builder.AddViews();
+            AddRazorViewEngineFeatureProviders(builder);
             AddRazorViewEngineServices(builder.Services);
             return builder;
         }
 
+        private static void AddRazorViewEngineFeatureProviders(IMvcCoreBuilder builder)
```

Comment:
Move down. Wrong ordering :smile: [link](http://stackoverflow.com/questions/150479/order-of-items-in-classes-fields-properties-constructors-methods)